c/bindTest.c

```diff
@@ -0,0 +1,103 @@
+/*
+  This program and the accompanying materials are
+  made available under the terms of the Eclipse Public License v2.0 which accompanies
+  this distribution, and is available at https://www.eclipse.org/legal/epl-v20.html
+  
+  SPDX-License-Identifier: EPL-2.0
+  
+  Copyright Contributors to the Zowe Project.
+*/
+
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <stdarg.h>
+
+#include "utils.h"
+#include "bpxnet.h"
+#include "le.h"
+
+/* returns valid */
+static int validateAddress(char *address, InetAddr **inetAddress) {
+  *inetAddress = getAddressByName(address);
+  if (!strcmp(address,"0.0.0.0")) {
+    return TRUE;      
+  }
+  if (!(*inetAddress && (*inetAddress)->data.data4.addrBytes)) {
+    printf("Error: Could not resolve hostname from DNS. Is this a valid hostname for this system?\n");
+    return FALSE;
+  } else {
+    int part4 = 0xff & (*inetAddress)->data.data4.addrBytes;
+    int part3 = (0xff00 & (*inetAddress)->data.data4.addrBytes) >> 8;
+    int part2 = (0xff0000 & (*inetAddress)->data.data4.addrBytes) >> 16;
+    int part1 = (0xff000000 & (*inetAddress)->data.data4.addrBytes) >> 24;
```

Comment:
IPv6 support is coming soon, so I've filed https://github.com/zowe/zss/issues/770 as a reminder to enhance this at that time.